scripts/data/updateWidgetsOverview.ts

```diff
@@ -43,9 +43,12 @@ async function main(): Promise<void> {
                         }
                     );
 
-                    const hasPreview = await fs
+                    const hasStructurePreview = await fs
                         .readdir(join(packagePath, "src"))
                         .then(files => files.some(path => path.endsWith(".editorConfig.ts")));
```

Comment:
Checking editorConfig file does not guarantee we have structure preview, it might be the case the file is just being used to create conditional properties, so better check if getPreview returns something else besides null.

Reply:
fair point. I now run the files through the typescript checker to see if the appropriate function is exported.
Checking that it returns something other than null seems tricky; not sure if that's feasible, and at this point, I think it's not really worth it.